department-of-veterans-affairs/caseflow
# Pass date and docket number in controller and update UI after submit

Resolves [CASEFLOW-686](https://vajira.max.gov/browse/CASEFLOW-686)

### Description
Updates the appeals controller to pass the new docket number and new NOD date when the NOD date gets updated for a case. Updates the frontend to display the new docket number immediately on successful submit.

### Acceptance Criteria

### Testing Plan
1. Switch to any user that can edit NOD date
2. Go to an AMA appeal
3. Update NOD date
4. Ensure the docket number updates after submit
